values.yaml

```diff
@@ -183,7 +183,7 @@ env:
   REDIS_SENTINEL_MASTER_NAME: ""
   REDIS_SENTINELS: ""
   REDIS_HOST: 15.206.207.127
```

Comment:
could you please remove the hardcode redis host name ?

Reply:
Done!